### PR TITLE
Removing 'Changing your email will require confirming the new email' hint from signup page.

### PR DESCRIPTION
**Pull request by f3rmat**
Fixes:  #3665

### CODE REVIEW

**Comment by f3rmat** [comment]
@viroulep 
Any suggestions about this change?

File: WcaOnRails/app/views/users/edit.html.erb [hunk 1]
```diff
@@ -143,6 +143,7 @@
         <%= simple_form_for @user do |f| %>
           <%= hidden_field_tag "section", "email" %>
           <%= f.input :email, disabled: !editable_fields.include?(:email) %>
```
Comment (jfly) [inline WcaOnRails/app/views/users/edit.html.erb]: If you change this line as follows:

```suggestion
           <%= f.input :email, disabled: !editable_fields.include?(:email), hint: t('.confirm_new_email') %>
```

Can you get rid of the `span.help-block` you added, along with the CSS you added above?

File: WcaOnRails/app/assets/stylesheets/users.scss [hunk 2]
```diff
@@ -42,3 +42,7 @@ input[type="radio"].center-vertically {
 #avatar-img {
   max-height: 400px;
 }
+
+#email .help-block {
```
Comment (jfly) [inline WcaOnRails/app/assets/stylesheets/users.scss]: If my suggestion on `edit.html.erb` works, then I think we can get rid of this css.

File: WcaOnRails/app/views/users/edit.html.erb [hunk 3]
```diff
@@ -142,7 +142,7 @@
       <div class="tab-pane active" id="email">
         <%= simple_form_for @user do |f| %>
           <%= hidden_field_tag "section", "email" %>
-          <%= f.input :email, disabled: !editable_fields.include?(:email) %>
+           <%= f.input :email, disabled: !editable_fields.include?(:email), hint: t('.confirm_new_email') %>
```
Comment (jfly) [inline WcaOnRails/app/views/users/edit.html.erb]: It looks like you added an extra space here. I'll remove it for you.

**Comment by jfly** [comment]
Merged and deployed! Ahh, that looks much nicer:

image

Thanks for the contribution, @f3rmat! :chart_with_upwards_trend:

**Pull request merged**